DisplaySettings/DisplaySettings.json

```diff
@@ -1132,7 +1174,7 @@
             }
         },
         "getSurroundVirtualizer":{
-            "summary": "Returns the current surround virtualizer boost settings",
+            "summary": "(Version 1) Returns the current surround virtualizer boost settings.\n \n### Event \n\n No Events.",
```

Comment:
The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

Reply:
As discussed keeping only version2 methods in md file.